test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -440,6 +464,18 @@ def get_paths(self):
                     paths['memsw'] = paths['memory']
         return paths
 
+    def is_dir(self, cpath, mom):
+        """
+        Returns True if path exists otherwise false
+        """
+        x = 0
+        while (x < 10):
```

Comment:
You should use...

for x in range(10):

Since you don't actually use the value of x, you can also do the following:

for _ in range(10):